portal-impl/src/com/liferay/portal/template/ServiceLocator.java

```diff
@@ -41,6 +66,12 @@ public Object findService(String servletContextName, String serviceName) {
 	private ServiceLocator() {
 	}
 
+	private static final boolean _TEMPLATE_ENGINE_SERVICE_LOCATOR_RESTRICT =
```

Comment:
@tinatian this is in portal-impl, the props should be declared in PropsValues. Please add a field there.

Reply:
@shuyangzhou It is only used once in this class, why have to declare in PropsValues? I remember we do have properties without definition in PropsValues.

Reply:
We normally only do that when usage is inside kernel. It is not a strictly enforced rule. There might be exceptions. But when the usage is inside impl, keep it inside PropsValues to follow the common style.